spec/services/booked_visits_csv_exporter_spec.rb

```diff
@@ -40,6 +40,7 @@
       expect(row['Prisoner number']).to eq(booked_visit.prisoner_number)
       expect(row['Slot granted']).
         to eq(instance.format_slot_for_staff(booked_visit.slot_granted))
+      expect(row['Closed visit']).to be_nil
```

Comment:
I think it would be better to keep it consistent with the visitor `_allowed?` methods and return `true` or `false` instead of `nil`